testing/rdiffbackupdeletetest.py

```diff
@@ -0,0 +1,144 @@
+import os
+import sys
+
+from distutils import spawn
+import subprocess
+import unittest
+
+PY2 = sys.version_info < (3,)
+PY3 = sys.version_info > (3,)
+
+# Working directories
+old_test_dir = os.path.join(os.path.dirname(os.getcwd()), 'rdiff-backup_testfiles')

```

Comment:
old_test_dir is already defined in commontests.py

Reply:
Fixed